File: src/main/java/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/domainarchitecturemapping/DomainArchitectureMappingObject.java
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

import java.util.LinkedHashSet;

@Getter
@ToString
public class DomainArchitectureMappingObject {

    @JsonProperty("dq_id")
    private String dqID;
    private String name;
    @JsonProperty("operation_id")
    private String operationID;
    private String type;
    @JsonProperty("implements")
    private LinkedHashSet<String> implementations;
    private LinkedHashSet<String> objects;
    private LinkedHashSet<Activity> activities;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/domainarchitecturemapping/DomainArchitectureMappingObject.kt
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class DomainArchitectureMappingObject {
    @JsonProperty("dq_id")
    val dqID: String? = null
    val name: String? = null

    @JsonProperty("operation_id")
    val operationID: String? = null
    val type: String? = null

    @JsonProperty("implements")
    val implementations: LinkedHashSet<String>? = null
    val objects: LinkedHashSet<String>? = null
    val activities: LinkedHashSet<io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping.Activity>? = null
}
